package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "mongo-mongoose",
+  "version": "1.0.0",
+  "description": "",
+  "main": "mongoosePlayground.js",
+  "dependencies": {
+    "dotenv": "^6.2.0",
+    "express": "^4.16.4",
+    "mongoose": "^5.4.6",
+    "morgan": "^1.9.1",
+    "nodemon": "^1.18.9",
+    "superagent": "^4.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^5.12.1",
+    "jest": "^23.6.0",
+    "supertest": "^3.4.2"
+  },
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch --runInBand",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "start": "nodemon server.js"
+  },
```

```suggestion
  },
  "jest": {
    "testEnvironment": "node"
  },
```
Jest will warn you during testing if the environment is not set up correctly. Either add to package.json or add a jest.config.js with appropriate settings:
https://jestjs.io/docs/en/configuration